tests/ReviewsApiTest.php

```diff
@@ -13,54 +13,42 @@ class ReviewsApiTest extends TestCase
     protected $reviewsApiUrl = 'api/v2/reviews';
 
     /**
-     * @todo - The reviews endpoint needs to be updated to work
-     * with the new schema and this test will have to be rewritten
-     * when that happens.
-     *
-     * Test that a PUT request to /reviews updates the status and creates a new event and review.
+     * Test that a PUT request to /reviews updates the post's status and creates a new event and review.
      *
      * PUT /reviews
      * @return void
      */
-    // public function testUpdatingASingleReview()
-    // {
-    //     // Create a photo.
-    //     $photo = factory(Photo::class)->create();
+    public function testUpdatingASingleReview()
+    {
 
-    //     // Create a post and associate the photo with it.
-    //     $post = factory(Post::class)->create([
-    //         'postable_id' => $photo->id,
-    //         'postable_type' => 'photo',
-    //     ]);
-    //     $post->content()->associate($photo);
-    //     $post->save();
+        // Create a post.
+        $post = factory(Post::class)->create();
 
-    //     $review = [
-    //         'rogue_event_id' => $post->event_id,
-    //         'status' => $this->faker->word(),
-    //         'event_type' => 'review_photo',
-    //         'reviewer' => str_random(24),
-    //     ];
+        $review = [
+            'post_id' => $post->id,
+            'status' => $this->faker->word(),
```

Comment:
I thought that we had validation to make sure that the Post status was `accepted` or ` rejected` or `pending`. Does this not give a validation error because it comes in through the `/reviews` endpoint instead of the `/posts` one?